src/com/projectkorra/projectkorra/util/DamageHandler.java

```diff
@@ -16,6 +16,7 @@
 import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 import org.bukkit.event.entity.EntityDamageEvent.DamageModifier;
 
+@SuppressWarnings("deprecation")
```

Comment:
Firstly, it is better to have the actual suppression where the warning actually is. Secondly, we want to avoid suppressions altogether.